src/AzSK.ADO/Framework/Abstracts/ADOSVTBase.ps1

```diff
@@ -4,6 +4,7 @@ class ADOSVTBase: SVTBase {
 	hidden [AzSKSettings] $AzSKSettings;
 	# below variable will be used by SVT's and overriden for each individual resource.
 	hidden [bool] $isResourceActive = $true;
+	hidden [string] $ResourceLink;
```

Comment:
purpose of adding ResourceLink in ADOSVTBase ?

Reply:
ADOSVTBase.ps1 file is inherited by all other SVT's thats why this gobal "ResourceLink" variable here.